src/_assets/css/main.scss

```diff
@@ -628,19 +628,6 @@ img {
   }
 }
 
-
-/* -----------------------------------------
- Dartpad
- ----------------------------------------- */
-
-#run-dartpad {
-  position: relative;
-  padding: 15px 0;
-  background: $gray-light;
-  border-top: 1px solid $gray;
-  z-index: 5;
-}
-
```

Comment:
This CSS class was introduced July 11, 2016, [Migrating repo from Ocupop](https://github.com/dart-lang/site-www/commit/fde624e85deceb0590beeb5ab65a06f8e31a1ffa). AFAICT this style isn't used anymore (I've checked the site sources and the generated site).